Patch to fix compilation with the Intel compiler. It replaces some 'use'
statements with 'import' in several interfaces.
author: Micael Oliveira (Max Planck Institute for the Structure and Dynamics of Matter)
diff -rup futile-1.8.3.orig/flib/fpython.f90 futile-1.8.3/flib/fpython.f90
--- futile-1.8.3.orig/flib/fpython.f90	2019-08-29 17:04:28.319733800 +0200
+++ futile-1.8.3/flib/fpython.f90	2019-08-29 17:04:34.895795372 +0200
@@ -1,6 +1,7 @@
 module f_python
   use f_precisions, only: f_address
-
+  use dictionaries
+  
   implicit none
 
   !> Equivalent type than the numpy one, to be able
@@ -24,7 +25,7 @@ module f_python
 
   interface
      subroutine f_python_initialize(iproc, nproc, igroup, ngroup)
-       use dictionaries
+       import
        implicit none
        integer, intent(in) :: iproc, nproc, igroup, ngroup
      end subroutine f_python_initialize
@@ -32,14 +33,14 @@ module f_python
 
   interface
      subroutine f_python_finalize()
-       use dictionaries
+       import
        implicit none
      end subroutine f_python_finalize
   end interface
 
   interface
      subroutine f_python_execute_dict(dict, status)
-       use dictionaries
+       import
        implicit none
        type(dictionary), pointer :: dict
        integer, intent(out) :: status
@@ -48,7 +49,7 @@ module f_python
 
   interface
      subroutine f_python_execute(script, status)
-       use dictionaries
+       import
        implicit none
        character(len = *), intent(in) :: script
        integer, intent(out) :: status
